qubes-vm/qmemman/qmemman-4.0.22.ebuild

```diff
@@ -40,4 +40,5 @@ install_systemd_units() {
 src_install() {
 	dosbin meminfo-writer
 	install_systemd_units "${SYSTEMD_UNITS[@]}"
+	newinitd "${FILESDIR}"/qubes-meminfo-writer.initd qubes-meminfo-writer
```

Comment:
let's copy function from core-agent and use that for consistency. Later we will go through common functionality and put that together in an eclass or sth.